Assignment2/manumazvi-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,44 @@
+{	
+	"@context":{
+		"vocab":"http://example.com/semweb/",
+		"name":"vocab:name",
+		"hometown":"vocab:hometown",
+		"age":"vocab:age",
+		"almaMater":"vocab:almaMater",
+		"parentsName":"vocab:parentsName",
+		"subjects":"vocab:subjects",
+		"name":"subjects:name",

```

Comment:
You have a duplicate name. Rename it, maybe as "subject":"vocab:subject" according to the name provided at the structure of your subject's array.